lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -429,6 +429,25 @@ push @tests, __PACKAGE__->new(
 );
 
 
+# They should be multiple selected from account_link_description(description)
```

Comment:
This change does not make sense and does not belong in this change: accounts are not required to have links and whether or not they do is unrelated to reconciled balances